doc/source/formats/format_overview.txt

```diff
@@ -0,0 +1,41 @@
+====================================  =========================================  ==========  =============  ==============
+File type                             Description                                Topology    Coordinates    Read/Write
```

Comment:
I would make it two columns: Read | Write and use Yes/No or just a [checkmark](https://en.wikipedia.org/wiki/Check_mark) ✓ for anything that works and leave empty or dash – for missing functionality.